modules/lead/product-base/variables.tf

```diff
@@ -3,3 +3,6 @@ variable "product_name" {}
 variable "image_whitelist" {}
 variable "vault_namespace" {}
 variable "vault_root_token_secret" {}
+variable "vault_path_url" {
+    default = ""
```

Comment:
why not set the default to `http://vault.${var.vault_namespace}.svc.cluster.local` since that's effectively how you're using it?

Reply:
This is a good point and I just pushed those changes so I'll mark this as resolved.